solutions/banking/watson-scripts/assistant-read.sh

```diff
@@ -2,9 +2,21 @@
 
 set -e
 
-project_name="gen-ai-rag-sample-app-assistant"
 token=$(curl -fLsS -X POST 'https://iam.cloud.ibm.com/identity/token' -H 'Content-Type: application/x-www-form-urlencoded' -d "grant_type=urn:ibm:params:oauth:grant-type:apikey&apikey=$API_KEY" | jq -r '.access_token')
 
-curl -X GET --location "$WATSON_ASSISTANT_URL/v2/assistants?version=2023-06-15" \
+IN=$(cat)
+EXISTING_ASSISTANT_ID=$(echo "$IN" | jq -r .assistant_id)
```

Comment:
Add a safety check in case the assistant_id is empty. Also check for the response and exit 1 (or return empty data) if for some reason the assistant ID is not found - unless that API does return 404 and it terminates the script.